data-raw/tidyverse_creation.R

```diff
@@ -0,0 +1,177 @@
+library(dplyr)
+# ------------------------------------------------------------------------------
+body_function <- function(fct, argument, args1) {
+
+  body_arg <- paste0("(", paste(args1, collapse = ", "), ", ...)")
+  body_f <- NULL
```

Comment:
This should be suppressed if the code is reviewed according to the suggestions below.

Reply:
Done